### PR TITLE
fix: empty initial search value

### PR DESCRIPTION
**Pull request by MaxPostema**
fixes: molgenis/molgenis-frontend#671

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
-  User documentation updated
- [x] Clean commits
-  No warnings during install
-  Updated flow typing

### CODE REVIEW

**Comment by fdlk** [comment]
This only worked for the initial select, but the underlying component is simply buggy.
E.g. If you search for "label4", then select it, then remove it using the cross, and open the search window again the search box still reads "label4".

New fix switches to a different component that does not have these issues.

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <script src="node_modules/es6-shim/es6-shim.min.js"></script>
     <script src="https://unpkg.com/@molgenis/expressions"></script>
     <link rel="shortcut icon" href="https://avatars1.githubusercontent.com/u/1688158?v=4&s=280" />
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://preview-theme-pr-91.dev.molgenis.org/%40molgenis-ui/molgenis-theme/dist/themes/mg-molgenis-blue-4.css" crossorigin="anonymous">
```
Comment (MaxPostema) [inline index.html:10]: Should there be a preview url here?

File: src/components/field-types/MultiSelectFieldComponent.vue [hunk 2]
```diff
@@ -140,9 +165,15 @@ export default {
...
-    vSelect,
+    Multiselect,
     FormFieldMessages,
     Description
   }
 }
 </script>
+<style scoped>
+  .multiselect {
+    width: 1%;
```
Comment (MaxPostema) [inline src/components/field-types/MultiSelectFieldComponent.vue:176]: `width: 1%;` feels a bit strange, is there an important reason for this?

Comment (MaxPostema) [reply src/components/field-types/MultiSelectFieldComponent.vue:176]: I see the reason now. Feels like the parent element needs some `flex-wrap: nowrap;` but i don't see a nice selector for that.

**Review by fdlk — APPROVED** [review]

**Pull request merged**